pipenv/utils.py

```diff
@@ -529,18 +522,17 @@ def resolve_deps(deps, which, which_pip, project, sources=None, verbose=False, p
             resolved_tree = actually_resolve_reps(deps, index_lookup, markers_lookup, project, sources, verbose, clear, pre)
         except RuntimeError:
             # Don't exit here, like usual.
-            resolved_tree = None
+            pass
 
     # Second (last-resort) attempt:
-    if resolved_tree is None:
-        with HackedPythonVersion(python_version='.'.join([str(s) for s in sys.version_info[:3]]), python_path=backup_python_path):
```

Comment:
We should keep the '.' and indentation fixes though

Reply:
That's fine,  let's get a PR up that pulls out what we need to remove.